package.json

```diff
@@ -134,7 +134,7 @@
     "ts-jest": "28.0.8",
     "ts-node": "10.9.1",
     "ts-toolbelt": "^9.6.0",
-    "typescript": "4.8.4",
+    "typescript": "4.9.4",
```

Comment:
I'll check tonight why my e2e test is failing

Reply:
I will split this into a separate PR as well to isolate it